mrp_partial_production/models/mrp.py

```diff
@@ -40,13 +44,16 @@ def get_qty_available_to_produce(self):
         lines = self.bom_id.explode(self.product_id, quantity)[1]
 
         result, lines_dict = defaultdict(int), defaultdict(int)
-        for res in self.move_raw_ids:
-            result[res.product_id.id] += res.reserved_availability
+        for res in self.move_raw_ids.filtered(
+                lambda move: move.is_done is False):
```

Comment:
Using "not" is faster than "is"
```python
self.move_raw_ids.filtered(lambda a: not a.is_done)
```
![image](https://user-images.githubusercontent.com/5271213/40330124-8d5ebb78-5d11-11e8-9937-59ac70e98f65.png)

Example:
![image](https://user-images.githubusercontent.com/5271213/40330411-76080faa-5d12-11e8-852d-dc66f287a8dc.png)

P.D. @moylop260  I love you

Reply:
Change applied Thanks @josemoralesp